docs/src/modules/java/pages/ask-akka/session.adoc

```diff
@@ -0,0 +1,267 @@
+= Modeling the agent session
+
+include::ROOT:partial$include.adoc[]
+
+Communication with an LLM is _stateless_. Everything that you get back from a model like ChatGPT is directly related to the prompt you submit. If you want to maintain a conversation and its history, then you have to do that work on your own in your agent code. With a few exceptions, the hosted  LLMs won't maintain this history for you.
+
+So before we get into the details of how to talk to an LLM, how to do document indexing, how to generate prompts, and many other details, let's get started with something simple like building a session (e.g. conversation) entity.
+
+== Overview
+
+This guide is concerned just with the session entity. We will get to more AI-specific tasks in the next steps.
+
+In this guide you will:
+
+* Create a new, empty Akka project
+* Create a data type for session events
+* Create a session entity
+* Create a conversation history view
+
+== Prerequisites
+
+include::ROOT:partial$cloud-dev-prerequisites.adoc[]
+
+== Create the empty project
+
+You already learned how to create an empty Akka project when you went through the guide to xref:java:author-your-first-service.adoc#_generate_and_build_the_project[author your first service]. Follow those steps again to ensure that you've got a new project that compiles.
+
+While you can use any settings you like, the code samples use:
+
+* Group ID: **akka.ask**
+* Artifact ID: **ask-akka**
+* Version: **1.0-SNAPSHOT**
+
+[NOTE]
+====
+This guide is written assuming you will follow it as a tutorial to walk through all of the components, building them on your own. If at any time you want to compare your solution with the official sample, check out the link:https://github.com/akka-samples/ask-akka-poc[Github repository].
```

Comment:
we should probably rename that repo, remove the `-poc`. I'd suggest `ask-akka-agent`